frontend/package.json

```diff
@@ -46,14 +45,12 @@
     "@typescript-eslint/eslint-plugin": "^5.56.0",
     "@typescript-eslint/parser": "^5.55.0",
     "babel-jest": "^29.4.2",
-    "chokidar-cli": "^3.0.0",
     "eslint": "^8.37.0",
     "eslint-config-next": "^13.3.0",
     "eslint-plugin-jest-dom": "^4.0.3",
     "eslint-plugin-testing-library": "^5.10.2",
     "fast-check": "^3.7.0",
     "husky": "^8.0.3",
-    "identity-obj-proxy": "^3.0.0",
```

Comment:
question (non-blocking): How did you know these were not in use? Did you use a library to detect this?